src/regular-grid-interpolator-implementation.cpp

```diff
@@ -245,17 +246,25 @@ std::vector<double> RegularGridInterpolatorImplementation::get_grid_point_data_r
 std::vector<Method> RegularGridInterpolatorImplementation::get_interpolation_methods() const
 {
     std::vector<Method> interpolation_methods(number_of_grid_axes);
+    static const std::unordered_map<InterpolationMethod, Method> interpolation_method_map {
```

Comment:
I'm not sure anymore that this translation has much value! Internally, everything is separated into InterpolationMethods and ExtrapolationMethods anyway...

Reply:
I believe it is needed to avoid further branching on interpolation vs. extrapolation in the performance critical code...but I haven't dug too deeply into it.

Reply:
OK, I'll mark this as ready for merge and we can add an issue to check this... someday. :)